<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="UI_ExtremaLinePoint_label_2">
         <property name="text">
          <string>Line(s) selected will be cut in </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="UI_CenterLinePoint_spin_numberLinePart">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The number indicates in how many &lt;span style=&quot; font-weight:600;&quot;&gt;Parts&lt;/span&gt; each selected parent Lines(s) will be cut in (Max 100).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="UI_ExtremaLinePoint_label_3">
         <property name="text">
          <string>parts</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="UI_ExtremaLinePoint_label">
           <property name="text">
            <string>Point(s) will be located at</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="UI_CenterLinePoint_checkBox">
           <property name="text">
            <string>all part's ends</string>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="UI_ExtremaLinePoint_label_4">
           <property name="text">
            <string>or at part's end number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="UI_CenterLinePoint_spin_indexPart">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The location of the point : 1/2 means middle of the segment !&lt;/p&gt;&lt;p&gt;The number indicates at which &lt;span style=&quot; font-weight:600;&quot;&gt;part's end&lt;/span&gt; the point will be located.&lt;/p&gt;&lt;p&gt;- If the Number of parts is&lt;span style=&quot; font-weight:600;&quot;&gt; 2&lt;/span&gt; and Point at part's end &lt;span style=&quot; font-weight:600;&quot;&gt;1&lt;/span&gt;,&lt;/p&gt;&lt;p&gt;this means that the point will be located in the &lt;span style=&quot; font-weight:600;&quot;&gt;middle&lt;/span&gt; of the Line.&lt;/p&gt;&lt;p&gt;- If the Number of parts is &lt;span style=&quot; font-weight:600;&quot;&gt;2&lt;/span&gt; and Point at part's end &lt;span style=&quot; font-weight:600;&quot;&gt;2&lt;/span&gt;, &lt;/p&gt;&lt;p&gt;this means that the point will be located in the &lt;span style=&quot; font-weight:600;&quot;&gt;end&lt;/span&gt; of the Line.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;Negative value are allowed&lt;/p&gt;&lt;p&gt;Limits : [-1000:1000]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="minimum">
            <number>-1000</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
